### ISSUE TITLE
No pipeline bind point in vkCmdPushConstants

### ISSUE THREAD

**Issue by procedural** [KhronosGroup/Vulkan-Docs#1189]
https://github.com/KhronosGroup/Vulkan-Docs/blob/3fe2595/chapters/commonvalidity/draw_dispatch_common.txt#L60-L61

> a push constant value must have been set for the same pipeline bind point

`vkCmdPushConstants` doesn't take a pipeline bind point.

Related: https://github.com/KhronosGroup/Vulkan-Docs/issues/1129

**Comment by procedural** [comment]
https://github.com/KhronosGroup/Vulkan-Docs/issues/1129#issuecomment-589567242

**Comment by krOoze** [comment]
Right, "for the same pipeline bind point" seems odd. Probably meant "with appropriate `stageFlags`"?

**Comment by procedural** [comment]
@krOoze probably, yea

**Comment by jeffbolznv** [comment]
The WG discussed this issue and agree with the suggestion. Would you mind making a PR?
